src/sentry/consumers/__init__.py

```diff
@@ -260,6 +260,14 @@ def ingest_events_options() -> list[click.Option]:
             "consumer_type": "events",
         },
     },
+    "ingest-feedback-events": {
+        "topic": settings.KAFKA_INGEST_FEEDBACK_EVENTS,
+        "strategy_factory": "sentry.ingest.consumer.factory.IngestStrategyFactory",
+        "click_options": ingest_events_options(),
+        "static_args": {
+            "consumer_type": "feedback-events",
+        },
+    },
     "ingest-attachments": {
         "topic": Topic.INGEST_ATTACHMENTS,
```

Comment:
See how they use `Topic.` here?  Apply this pattern to `ingest-feedback-events`.